CMakeLists.txt

```diff
@@ -23,6 +23,15 @@ set(CMAKE_C_STANDARD 99)
 set(TOP_DIRECTORY ${CMAKE_SOURCE_DIR})
 set(BUILD_DIRECTORY ${CMAKE_BINARY_DIR})
 
+#
+# use rpath for locating installed libraries
+#
+set(CMAKE_SKIP_BUILD_RPATH  FALSE)
```

Comment:
```suggestion
set(CMAKE_SKIP_BUILD_RPATH FALSE)
```

Reply:
Please note that the option's value is `FALSE` by default: https://cmake.org/cmake/help/latest/variable/CMAKE_SKIP_BUILD_RPATH.html